From 0b12831ceb20fa9a6e4b3299602589ecea808ae8 Mon Sep 17 00:00:00 2001
From: Nikhil <nikhil@menghani.com>
Date: Wed, 18 Jun 2014 13:42:57 +0530
Subject: [PATCH] external skia patch in slim

---
 src/core/SkPaint.cpp | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/src/core/SkPaint.cpp b/src/core/SkPaint.cpp
index 0e709b7..ca19f22 100644
--- a/src/core/SkPaint.cpp
+++ b/src/core/SkPaint.cpp
@@ -87,7 +87,12 @@ SkPaint::SkPaint() {
 }
 
 SkPaint::SkPaint(const SkPaint& src) {
-    memcpy(this, &src, sizeof(src));
+#ifdef NEON_BLIT_ANTI_H
+     if (sizeof(src) == SIZE_OF_PAINT)
+         memcpy_paint_opt((int*)this, (int*)&src);
+     else
+#endif
+         memcpy((int*)this, (int*)&src, sizeof(src));
 
     SkSafeRef(fTypeface);
     SkSafeRef(fPathEffect);
-- 
1.9.1

